tools/tkn-results/cmd/root.go

```diff
@@ -57,7 +58,15 @@ func Root() *cobra.Command {
 				return err
 			}
 
-			params.Client = apiClient
+			params.ResultsClient = apiClient
+
+			logClient, err := client.DefaultLogClient(cmd.Context())
```

Comment:
Shouldn't we include the overrideApiAdr here as well?

nit: It would be nice to rename `client.DefaultClient` above to `client.DefaultResultsClient`, but that's something we can do later. I have a list of few cosmetic changes and can include that one too.

Reply:
Yes, let me rebase with the merged code and then add that.

Reply:
I changed it to `DefaultResultsClient`